gr-utils/bindtool/templates/generic_python_cc.mako

```diff
@@ -58,7 +58,25 @@ if overloaded:
   overloaded_str = '({} ({}::*)({}))'.format(fcn['return_type'],cls_name,', '.join([f['dtype'] for f in fcn_args]))
 %>\
 % if fcn['name'] != filter_val:
-        ${modvar if isfree else ""}${".def_static" if has_static and not isfree else ".def"}("${fcn['name']}",${overloaded_str}&${cls_name}::${fcn['name']},
+%       if fcn['return_type'] == 'PyObject *': 
+##        .def("${fcn['name']}",
+##        [](std::shared_ptr<text_msg> p) {
+##            return PyLong_AsLongLong(p->${fcn['name']}());
+##        },
+<%
+func = '\n\
+            [](std::shared_ptr<'+cls_name+'> p) {\n\
+               return PyLong_AsLongLong(p->'+fcn_name+'());\n\
+        }'
+%>
+%       else: ## No PyObject pointer 
+<%
+func = overloaded_str+'&'+cls_name+'::'+fcn_name
+%>        
+%       endif                    
+##        ${modvar if isfree else ""}${".def_static" if has_static and not isfree else ".def"}("${fcn['name']}",${overloaded_str}&${cls_name}::${fcn['name']},
```

Comment:
Can you please remove the commented line since the overloading is moved up into the statement above